README.md

```diff
@@ -30,86 +29,95 @@ Pop supports Gtk+ 3.22.x
 
  > Monospace: Fira Mono Regular 11
 
-
 ### Installation
 
 Pop is intended to be installed through the package manager. Packages for Pop are available in PPA:
-```
+
+```sh
 sudo add-apt-repository ppa:system76/pop
 sudo apt update
 sudo apt install pop-theme
 ```
+
 It's recommended to use the `pop-theme` metapackage, as this will pull in all components of the look. However, individual components can be installed separately, e.g:
-```
+
+```sh
 sudo apt install pop-gtk-theme
 ```
+
 It's also recommended to restart the GNOME Shell after applying the theme of your choice.
 
 Enter the Shell's command launcher
+
 ```
 Alt + F2
 ```
 
 This will restart the Shell after you hit Enter
+
 ```
 r
 ```
 
-
-
 ### Installation from Git Source
 ----------------------------
 
 This is the recommended method for users who aren't on Pop.
 
 ###### Note: You must have sassc installed in order to build Pop. Users of 17.04 or later can all build-dependencies using:
 
-```
-sudo apt install sassc meson libglib2.0-dev 
+```sh
+sudo apt install sassc meson libglib2.0-dev
 ```
 
 For making modifications to assets, you will additionally need these two:
 
-```
+```sh
 sudo apt install inkscape optipng
 ```
 
+**NB:** We do not guarantee that we will keep the set of commands below up to date nor support it. That said, to install dependencies on Fedora:
```

Comment:
I still don't think I'm comfortable specifically mentioning a distribution which Pop isn't based on, or with which it doesn't share a lineage. I'd prefer to list the generic dependencies in a more distro agnostic fashion without calling out to specific distributions like Fedora, Arch, etc.

The current build dependencies are:
```
glib-2.0
libsass >= 3.6
sassc >= 3.6
gtk-themes-extra
gtk2-engines-pixbuf
gtk2-engines-murrine
```
Please remove any direct references or specific build instructions for Fedora, and instead include a distro-agnostic list of dependencies. Retaining the instructions for installing these dependencies on Debian-based distributions is desirable.